lib/preprocessor.js

```diff
@@ -3,7 +3,6 @@
 const fs = require('graceful-fs')
 const mm = require('minimatch')
 const isBinaryFile = require('isbinaryfile')
-const _ = require('lodash')
```

Comment:
This refactoring is not needed for the 'can return Promise' feature correct?  
Can you move it to its own PR?  Otherwise it will be squashed with the feature and could make future debugging harder.

Reply:
The second commit will be in another PR